### ISSUE TITLE
overly late template markings when all suggestions are applied

### ISSUE THREAD

**Issue by AndiBraimllari** [OpenCovenant/quill#369]

**Comment by AndiBraimllari** [comment]
Is this fixed @vaqueraoscar0? Can you @AndersonCeci please test this, so we can close it if all seems fine.

**Comment by AndersonCeci** [comment]
looking fine,  looking good 👌

**Comment by vaqueraoscar0** [comment]
@AndiBraimllari Yeah it should fix the stated. Anything outside of this might still cause issues though, like mixing dismissed cards and suggested cards. i'll Work on the mixing issue but its going to take some time.

**Comment by AndiBraimllari** [comment]
Sure, take your time! I'm closing this issue then.
